Cargo.toml

```diff
@@ -9,7 +9,11 @@ license = "Apache-2.0"
 
 [features]
 default = ["tokio-net"]
-rustls = ["tokio-rustls"]
+rustls-core = ["tokio-rustls"]
+rustls-aws-lc = ["rustls-core", "tokio-rustls/aws-lc-rs"]
```

Comment:
All these features aren't strictly necessary. 

The user could potentially enable these features on the tokio-rustls dependency directly. 

All this create actually needs to depend on is the core tokio-rustls api.

Reply:
Well, you are right. But consider the fact that `tokio-rustls` which is also an "intermediate" crate between us and the actual `rustls` crate has similar feature flags which do nothing but pass themselves to `rustls`. (I've checked it's source code and they actually do nothing else)